java/src/jmri/BlockManager.java

```diff
@@ -101,16 +101,16 @@ public Block createNewBlock(@Nonnull String systemName, @CheckForNull String use
                 return null;
             }
         }
-        r = getBySystemName(systemName);
+        String sName = systemName.toUpperCase();
```

Comment:
Please don't explicitly "toUpperCase" system names like this.  That ends up with pieces of inconsistent code in various places. Instead, please put that into the `normalizeSystemName(@Nonnull String inputName)` method.